polling_stations/apps/data_collection/management/commands/import_mole_valley.py

```diff
@@ -3,13 +3,15 @@
 
 class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E07000210"
-    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019.tsv"
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019Mole.tsv"
+    stations_name = "parl.2019-12-12/Version 1/Democracy_Club__12December2019Mole.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
+    csv_encoding = "windows-1252"
+    allow_station_point_from_postcode = False
 
     def station_record_to_dict(self, record):
-        # Carried over from local.2019-05-02
+        # Carried over from local.2019-05-02 & europarl.2019-05-23
         if record.polling_place_id == "3742":  # Catholic Church Hall
```

Comment:
There is no station id 3742 in this file
If the same station is in use, the ID will need updating